inc/class-sitemaps.php

```diff
@@ -1366,7 +1362,7 @@ private function get_datetime_with_timezone( $datetime_string ) {
 			$datetime = new DateTime( $datetime_string, $utc_timezone );
 			$datetime->setTimezone( $local_timezone );
 
-			return $datetime;
+			return $datetime->format( $format );
```

Comment:
This breaks backwards compat by changing signature of the function, PHPDoc is outdated accordingly.